packages/shared/src/components/post/PostActions.tsx

```diff
@@ -166,14 +172,30 @@ export function PostActions({
             Bookmark
           </QuaternaryButton>
           <QuaternaryButton
-            id="share-post-btn"
+            id="copy-post-btn"
             onClick={() => onCopyLinkClick(post)}
             icon={<LinkIcon />}
             responsiveLabelClass={actionsClassName}
-            className="btn-tertiary-cabbage"
+            className={classNames(
+              'btn-tertiary-cabbage',
```

Comment:
IIRC we should be able to send `variant` and `color` props instead. Feel free to ignore if my memory fails me 😊

Reply:
Oh, right. Didn't notice this. Lol. Thank you.